.ci/run-with-credentials.sh

```diff
@@ -39,7 +39,7 @@ clirr)
   mvn clirr:check
   ;;
 integration)
-  mvn verify -Dclirr.skip=true -DskipITs=false
+  mvn verify -Dclirr.skip=true -DskipITs=false -DPG_ADAPTER_HOST="https://${GOOGLE_CLOUD_ENDPOINT}" -DPG_ADAPTER_INSTANCE="${GOOGLE_CLOUD_INSTANCE}" -DPG_ADAPTER_DATABASE="${GOOGLE_CLOUD_DATABASE}"
```

Comment:
The CI script sets a number of environment variables that should be used as the endpoint, instance, database etc. Those environment variables were not used by the integration tests, as the integration tests expect those to be given as system properties and not as environment variables. This ensures that the values that have been configured are actually fed in to the integration tests as system properties.